src/classes/UTIL_Describe.cls

```diff
@@ -112,6 +113,36 @@ public class UTIL_Describe {
             }
         }
     }
+	
+    /*******************************************
+    * Gets a list of picklist entries for a new object/field/list<PicklistEntry> triple
+    ********************************************/
+    private static void fillPicklistEntriesForObject(string objectName, string fieldName) {
+		
+		// make sure we have this field's schema mapped
+        if (!fieldDescribes.containsKey(objectName) || !fieldDescribes.get(objectName).containsKey(fieldName))
+            fillFieldMapsForObject(objectName, fieldName);
+				
+		// Check for picklist.
+		if(getFieldType(objectName,fieldName) != 'PICKLIST') {
+			throw new SchemaDescribeException('Invalid Selection. Field name \'' + fieldName + '\' for object name \'' + objectName + '\' is not a picklist.');
+		}
+
+		// fill entries for the first time.
+		
+		//Create object to field map for picklist if needed.
+        if (!fieldPicklistEntries.containsKey(objectName)) {
+            fieldPicklistEntries.put(objectName, new Map<String, List<Schema.PicklistEntry>>());
+        }
+		
+		//Create object to picklist map if needed.
```

Comment:
More funky spacing here, just verify all is well in your editor. (I'm assuming this looks good in the editor and its just a "git" thing

Reply:
It's looking fine in my editor, but there were tabs in there. I took this opportunity to strip all tabs from the file and replace them with four spaces.